extensions/amp-jwplayer/0.1/amp-jwplayer.js

```diff
@@ -128,6 +132,15 @@ class AmpJWPlayer extends AMP.BaseElement {
 
   /** @override */
   isInteractive() {
+    if (
+      this.element.hasAttribute('autoplay') &&
+      isAutoplaySupported(this.win) &&
```

Comment:
This is incorrect as `isAutoplaySupported` returns a promise.

Reply:
Ah. Must have misread it. Updated it to use it correctly.